wanted-next-generation-software-engineer.md

```diff
@@ -2,27 +2,23 @@
 
 Hi There,
 
-Our company AutoScout24 GmbH (http://www.autoscout24.com/) is searching for Senior Engineer with relocation to Munich,
+Our company AutoScout24 GmbH (http://www.autoscout24.com/) is searching for a Senior Engineer with relocation to Munich,
```

Comment:
Then write `Senior Engineers`. Either `I want an apple` or `I want apples`, but not `I want apple`, I think.